pkg/store/consul/kv.go

```diff
@@ -28,8 +28,9 @@ import (
 const (
 	TTL = 60 * time.Second
 
-	// Don't change this, it affects where pod status keys are read and written from
+	// Don't change this, it affects where status keys are read and written from
 	PreparerPodStatusNamespace statusstore.Namespace = "preparer"
+	RCStatusNamespace          statusstore.Namespace = "rc_farm"
```

Comment:
I gotta nit the name because this will be hard to change. Avoid contractions, go for the explicit "replication_controller"

Reply:
Will do